generators/app/index.js

```diff
@@ -124,42 +121,13 @@ module.exports = class extends BaseGenerator {
         }
     }
 
-    install() {
-        let logMsg =
-            `To install your dependencies manually, run: ${chalk.yellow.bold(`${this.clientPackageManager} install`)}`;
-
-        if (this.clientFramework === 'angular1') {
-            logMsg =
-                `To install your dependencies manually, run: ${chalk.yellow.bold(`${this.clientPackageManager} install & bower install`)}`;
-        }
-        const injectDependenciesAndConstants = (err) => {
-            if (err) {
-                this.warning('Install of dependencies failed!');
-                this.log(logMsg);
-            } else if (this.clientFramework === 'angular1') {
-                this.spawnCommand('gulp', ['install']);
-            }
-        };
-        const installConfig = {
-            bower: this.clientFramework === 'angular1',
-            npm: this.clientPackageManager !== 'yarn',
-            yarn: this.clientPackageManager === 'yarn',
-            callback: injectDependenciesAndConstants
-        };
-        if (this.options['skip-install']) {
-            this.log(logMsg);
-        } else {
-            this.installDependencies(installConfig);
-        }
-    }
-
     end() {
         this.log('End of spring-cloud-stream generator');
     }
 
     installRabbitMq() {
         const STREAM_RABBIT_VERSION = '1.3.0.RELEASE';
-        const STREAM_CLOUD_DEPENDENCY_VERSION = 'Ditmars.RELEASE';
```

Comment:
It's a wrong verion here, isn't it ?

Reply:
I take this version on Spring documentation, I don't check the version use in jhipster..

Reply:
Oh, the version Ditmars really exists, I changed to the same version as JHipster: Chelsea
So here, I don't know which version we should use?

Reply:
stay consistent with the jhipster version :)